### PR TITLE
Make links match those we provide in Discord

### PR DESCRIPTION
**Pull request by jhaaaa**
- Per Fabri, let's make the links in the XMTP org README match the links we provide in Discord
- Updated page title to reflect the latest XMTP key messaging

### CODE REVIEW

**Review by nplasterer — APPROVED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
-# XMTP: the web3 messaging protocol
+# XMTP: Web3’s secure messaging network and protocol
 
-📚 Read [our docs](https://docs.xmtp.org) on how to start building with XMTP
+✨ Built with XMTP: https://xmtp.org/apps
```
Comment (galligan) [inline profile/README.md:3]:
```suggestion
✨ Built with XMTP: <https://xmtp.org/apps>
```

File: profile/README.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 
-💬 Check out the [Quickstart React chat app](https://github.com/xmtp/xmtp-quickstart-react), demonstrating core capabilities of the XMTP client SDK
+🥷🏻 Hackathons: https://xmtp.org/hackathons
```
Comment (galligan) [inline profile/README.md:5]:
```suggestion
🥷🏻 Hackathons: <https://xmtp.org/hackathons>
```

File: profile/README.md [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
 
-💬 Check out the [XMTP Inbox chat app](https://github.com/xmtp-labs/xmtp-inbox-web), demonstrating core and advanced capabilities of the XMTP client SDK
+📚 Tutorials: https://xmtp.org/tutorials
```
Comment (galligan) [inline profile/README.md:7]:
```suggestion
📚 Tutorials: <https://xmtp.org/tutorials>
```

File: profile/README.md [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
 
-💾 Explore our [JavaScript SDK](https://github.com/xmtp/xmtp-js) for building messaging experiences that use XMTP
+💨 Quickstart: https://xmtp.org/start
```
Comment (galligan) [inline profile/README.md:9]:
```suggestion
💨 Quickstart: <https://xmtp.org/start>
```

File: profile/README.md [hunk 5]
```diff
@@ -1,13 +1,11 @@
...
 
-📜 Read our [XMTP Improvement Proposals (XIPs)](https://github.com/xmtp/XIPs)
-
-🐦 Follow [@xmtp_](https://twitter.com/xmtp_) on Twitter
+📖 Docs: https://xmtp.org/docs
```
Comment (galligan) [inline profile/README.md:11]:
```suggestion
📖 Docs: <https://xmtp.org/docs>
```

**Review by galligan — CHANGES_REQUESTED** [review]
@jhaaaa great changes! Quick little adjustment: bare links in Markdown should be wrapped in brackets, to signal that it should be rendered as a link.

Why? Well, some front-ends optimistically render links, but not all of them. If the Markdown linter were here, it might have thrown the error for you. 🫡

**Pull request merged**